### PR TITLE
Update documentation on building XA on Windows

### PR DESCRIPTION
**Pull request by grendello**
Make changes to sections which run the preparation stage of the build to use
xaprepare instead of the old `/t:Prepare` target.

### CODE REVIEW

File: Documentation/building/windows/instructions.md [hunk 1]
```diff
@@ -18,15 +18,19 @@ MSBuild version 15 or later is required.
...
 
-        msbuild Xamarin.Android.sln /t:PrepareExternal
+        build-tools/xaprepare/xaprepare/Debug/xaprepare /s:PrepareExternalGitDependencies
 
     This will configure external proprietary components such as monodroid.
 
  6. In a [Developer Command Prompt][developer-prompt], prepare the project:
 
-        msbuild Xamarin.Android.sln /t:Prepare
+        build-tools/xaprepare/xaprepare/Debug/xaprepare
```
Comment (jonathanpeppers) [inline Documentation/building/windows/instructions.md]: My question would be, can we just make `msbuild Xamarin.Android.sln /t:Prepare` do this step? and then we don't have to add a new step to building Xamarin.Android? (or change the docs at all 😄)

Comment (jonathanpeppers) [reply Documentation/building/windows/instructions.md]: Let's see if we can just do this instead: https://github.com/xamarin/xamarin-android/pull/3185

And if it doesn't work our for some reason, we can revisit this PR.

Comment (pjcollins) [reply Documentation/building/windows/instructions.md]: We should definitely be able to wrap whatever `xaprepare.exe` needs to do in the `/t:Prepare` target.

Comment (grendello) [reply Documentation/building/windows/instructions.md]: `Prepare` already performs the step, I updated this PR accordingly

File: Documentation/building/windows/instructions.md [hunk 2]
```diff
@@ -18,21 +18,15 @@ MSBuild version 15 or later is required.
 
  4. (Optional) [Configure the build](../configuration.md).
 
- 5. (Optional) In a [Developer Command Prompt][developer-prompt], prepare external git dependencies:
-
-        msbuild Xamarin.Android.sln /t:PrepareExternal
-
-    This will configure external proprietary components such as monodroid.
```
Comment (jonathanpeppers) [inline Documentation/building/windows/instructions.md]: I don't know if we need to remove these docs here, we still have these targets: https://github.com/xamarin/xamarin-android/blob/8cd3c1544c1f404d0c72d11462a493abece60d1c/build-tools/scripts/PrepareWindows.targets#L14-L17

I usually run `/t:PrepareExternal` to get `monodroid` on my machine.

We have the `Makefile` equivalent, too: https://github.com/xamarin/xamarin-android/blob/8cd3c1544c1f404d0c72d11462a493abece60d1c/Makefile#L238-L239

Comment (jonpryor) [reply Documentation/building/windows/instructions.md]: I agree that we shouldn't remove this section of the docs.

**Comment by grendello** [comment]
This PR is no longer necessary as building on Windows uses the same steps as before.

**Pull request closed without merging**